src/EmbeddedSonarAnalyzer.props

```diff
@@ -6,7 +6,7 @@
     <!-- Note: Guide on how to update the analyzers is on the xtranet! -->
     <EmbeddedSonarAnalyzerVersion>8.55.0.65544</EmbeddedSonarAnalyzerVersion>
     <EmbeddedSonarCFamilyAnalyzerVersion>6.43.0.61486</EmbeddedSonarCFamilyAnalyzerVersion>
-    <EmbeddedSonarJSAnalyzerVersion>10.0.1.20755</EmbeddedSonarJSAnalyzerVersion>
+    <EmbeddedSonarJSAnalyzerVersion>10.1.0.21143</EmbeddedSonarJSAnalyzerVersion>
```

Comment:
#3972 - update to the new version of the analyzer.

To test the change, I added JS/TS files to the sample project that test a new rule that was added in the new analyzer. Issues are raised, so it looks like the correct files are being extracted from the jar and embedded in the VSIX.

![image](https://user-images.githubusercontent.com/10757559/229827835-b711f50d-91dd-47c6-91fd-3b0e0e164007.png)